test/jdk/ProblemList.txt

```diff
@@ -581,25 +581,36 @@ javax/management/remote/mandatory/connection/RMIConnector_NPETest.java 8267887 g
 
 # jdk_net
 
+java/net/DatagramSocket/DatagramSocketExample.java              8308807 aix-ppc64
+java/net/DatagramSocket/DatagramSocketMulticasting.java         8308807 aix-ppc64
+
+java/net/MulticastSocket/B6427403.java                          8308807 aix-ppc64
+java/net/MulticastSocket/IPMulticastIF.java                     8308807 aix-ppc64
+java/net/MulticastSocket/JoinLeave.java                         8308807 aix-ppc64
+java/net/MulticastSocket/MulticastAddresses.java                8308807 aix-ppc64
 java/net/MulticastSocket/NoLoopbackPackets.java                 7122846 macosx-all
+java/net/MulticastSocket/NoSetNetworkInterface.java             8308807 aix-ppc64
+java/net/MulticastSocket/Promiscuous.java                       8308807 aix-ppc64
+java/net/MulticastSocket/SetGetNetworkInterfaceTest.java        8308807 aix-ppc64
 java/net/MulticastSocket/SetLoopbackMode.java                   7122846 macosx-all
-
+java/net/MulticastSocket/SetOutgoingIf.java                     8308807 aix-ppc64
 java/net/MulticastSocket/Test.java                              7145658 macosx-all
 
-java/net/MulticastSocket/SetGetNetworkInterfaceTest.java        8219083 windows-all
-
 java/net/ServerSocket/AcceptInheritHandle.java                  8211854 aix-ppc64
 
-
 ############################################################################
 
 # jdk_nio
 
-java/nio/channels/DatagramChannel/Unref.java                    8233437 generic-all
+
 
 java/nio/channels/AsynchronousSocketChannel/StressLoopback.java 8211851 aix-ppc64
 
+java/nio/channels/DatagramChannel/AdaptorMulticasting.java      8308807 aix-ppc64
+java/nio/channels/DatagramChannel/AfterDisconnect.java          8308807 aix-ppc64
```

Comment:
JDK-8316387 does not add AdaptorMulticasting.java.

Reply:
https://github.com/openjdk/jdk21u/commit/c86bad51a88b867de3cd0b1a50bf544c2111755f#diff-80063aaf72a19fa27f001c17fc7687086ea71b35b01f5e2b3fcd7f50d8c682fbR554

I am referring to this commit .